Base/res/ladybird/inspector.js

```diff
@@ -516,5 +552,45 @@ document.addEventListener("DOMContentLoaded", () => {
         event.preventDefault();
     });
 
+    document.addEventListener("keydown", event => {
+        const UP_ARROW_KEYCODE = 38;
+        const DOWN_ARROW_KEYCODE = 40;
+        const RETURN_KEYCODE = 13;
+        const SPACE_KEYCODE = 32;
+
+        if (document.activeElement.tagName !== "INPUT") {
+            if (event.keyCode == UP_ARROW_KEYCODE || event.keyCode == DOWN_ARROW_KEYCODE) {
+                let selectedIndex = visibleDOMNodes.indexOf(selectedDOMNode);
+                if (selectedIndex < 0) {
+                    return;
+                }
+
+                let newIndex;
+
+                if (event.keyCode == UP_ARROW_KEYCODE) {
+                    newIndex = selectedIndex - 1;
+                } else if (event.keyCode == DOWN_ARROW_KEYCODE) {
+                    newIndex = selectedIndex + 1;
+                }
+
+                if (visibleDOMNodes[newIndex]) {
+                    inspectDOMNode(visibleDOMNodes[newIndex]);
+                }
+            } else if (event.keyCode == RETURN_KEYCODE || event.keyCode == SPACE_KEYCODE) {
+                if (selectedDOMNode.parentNode.tagName !== "SUMMARY") {
+                    return;
+                }
+
+                let detailsElement = selectedDOMNode.parentNode.parentNode;
+
+                if (detailsElement.open) {
+                    detailsElement.removeAttribute("open");
+                } else {
+                    detailsElement.setAttribute("open", "");
```

Comment:
directly munging the "open" attribute of the details element feels a bit low level. Is there not some way we can trigger the activation behavior of the summary element?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLDetailsElement
Doesn't seem to be, but `click()` would probably work